sigs.yaml

```diff
@@ -361,6 +361,24 @@ sigs:
     contact:
       slack: sig-multicluster
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster
+  - name: GCP
+    dir: sig-gcp
+    mission_statement: >
+      A Special Interest Group for building, deploying, maintaining,
+      supporting, and using Kubernetes on the Google Cloud Platform.
+    leads:
+    - name: Adam Worrall
+      github: abgworrall
+      company: Google
+    meetings:
+    - day: Thursday
+      utc: "16:00"
+      frequency: monthly
```

Comment:
Aren't we meeting biweekly?

Reply:
Good catch, things changed since I drafted this :)